### PR TITLE
11777 update font to inter for calculator

### PR DESCRIPTION
**Pull request by shondriane**
### Ticket(s)

_Required unless this is just a change to documentation._

- _[https://airtable.com/appfJZShN8K4tcWHU/pagIxNNZ0BXdxESjz]_

### Type of change

_Please delete any options that are not relevant_

- [x] Design change

### Description

The calculator font was to be in Inter only not Sintony on the last page

Fixes # _replace with issue number that this fixes_

### Screenshots

_Include relevant screenshots_

#### Before
Screenshot 2024-02-21 at 4 56 37 PM

#### After
Screenshot 2024-02-21 at 4 55 39 PM

### Review your work
Please self review your work before requesting further review. _Wait until you reach the next screen to check off the boxes._

#### Preview
Use your local build.

- [x] Everything works as expected
- [x] Check changed page(s)
- [x] Click through calculator if changes were made
- [x] Compare figma screenshot with the changes
- [x] Compare linked Figma section to changes made
- [x] Refresh the page, check that using the go-back button works as expected
- [x] Check for any uncaught ESLint problems

### Overall PR Checklist:

_Please delete any options that are not relevant_

- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings in the console
- [x] There are no new linting errors/problems in my code
- [x] I have filled this PR out fully, and cleaned up any extraneous items so that it is clear and easy to understand

### CODE REVIEW

**Comment by emilyemilyemilyemilyemilyemily** [comment]
This is an automated comment I'm testing something. Logs:
```
 
```

**Comment by emilyemilyemilyemilyemilyemily** [comment]
This is an automated comment I'm testing something. Logs:
```
2024-02-22T01:05:52.930Z [INFO]: # Cloning repository: git@github.com:clearviction-devs/clearviction-wa.git
2024-02-22T01:05:57.873Z [INFO]: 
2024-02-22T01:05:57.969Z [INFO]: Cloning into 'clearviction-wa'...
2024-02-22T01:05:58.299Z [INFO]: From https://github.com/clearviction-devs/clearviction-wa
                                 * [new ref]         refs/pull/185/head -> pr-185
                                 Switched to branch 'pr-185'
2024-02-22T01:05:58.365Z [INFO]: Successfully cleaned up Git credentials
2024-02-22T01:05:58.365Z [INFO]: # Checking for Git submodules at: /codebuild/output/src4209441970/src/clearviction-wa/.gitmodules
2024-02-22T01:05:58.373Z [INFO]: # Retrieving environment cache...
2024-02-22T01:05:58.414Z [WARNING]: ! Unable to write cache: {"code":"ERR_BAD_REQUEST","message":"Request failed with status code 404"})}
2024-02-22T01:05:58.414Z [INFO]: ---- Setting Up SSM Secrets ----
2024-02-22T01:05:58.414Z [INFO]: SSM params {"Path":"/amplify/dqqxalj4ndxdp/prbif/","WithDecryption":true}
2024-02-22T01:05:58.459Z [WARNING]: !Failed to set up process.env.secrets
2024-02-22T01:06:02.894Z [INFO]: # Node version 18 is available for installation
2024-02-22T01:06:02.977Z [INFO]: # Installing Node version 18
2024-02-22T01:07:08.590Z [INFO]: # Now using Node version 18
2024-02-22T01:07:08.706Z [INFO]: # Retrieving cache...
2024-02-22T01:07:08.742Z [INFO]: # Retrieved cache
2024-02-22T01:07:13.126Z [INFO]: ## Starting Backend Build
                                 ## Checking for associated backend environment...
                                 ## No backend environment association found, continuing...
                                 ## Completed Backend Build
2024-02-22T01:07:13.137Z [INFO]: {"backendDuration": 0}
                                 ## Starting Frontend Build
                                 # Starting phase: preBuild
                                 # Executing command: npm ci
2024-02-22T01:07:14.041Z [WARNING]: npm
2024-02-22T01:07:14.041Z [WARNING]: WARN EBADENGINE Unsupported engine {
                                    npm WARN EBADENGINE   package: '@sanity/webhook@4.0.1',
                                    npm WARN EBADENGINE   required: { node: '>=20.0.0' },
                                    npm WARN
2024-02-22T01:07:14.042Z [WARNING]: EBADENGINE   current: { node: 'v18.19.1', npm: '10.2.4' }
                                    npm WARN EBADENGINE }
2024-02-22T01:07:49.919Z [INFO]: > clearviction-website@2.1.0 prepare
                                 > husky install
2024-02-22T01:07:49.966Z [WARNING]: install command is deprecated
2024-02-22T01:07:49.990Z [INFO]: added 937 packages, and audited 938 packages in 37s
2024-02-22T01:07:49.990Z [INFO]: 173 packages are looking for funding
                                 run `npm fund` for details
2024-02-22T01:07:49.992Z [INFO]: found 0 vulnerabilities
2024-02-22T01:07:50.026Z [INFO]: # Completed phase: preBuild
                                 # Starting phase: build
                                 # Executing command: npm run build
2024-02-22T01:07:50.314Z [INFO]: > clearviction-website@2.1.0 build
                                 > export build=true && next build
2024-02-22T01:07:50.867Z [INFO]: ⚠ No build cache found. Please configure build caching for faster rebuilds. Read more: https://nextjs.org/docs/messages/no-cache
2024-02-22T01:07:50.878Z [INFO]: Attention: Next.js now collects completely anonymous telemetry regarding usage.
2024-02-22T01:07:50.879Z [INFO]: This information is used to shape Next.js' roadmap and prioritize features.
                                 You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
                                 https://nextjs.org/telemetry
2024-02-22T01:07:50.960Z [INFO]: Linting and checking validity of types ...
2024-02-22T01:07:56.506Z [WARNING]: Failed to compile.
                                    ./components/layout/Header.tsx
                                    58:14  Error: `async          ` must be placed on a new line  react/jsx-one-expression-per-line
                                    59:1  Error: Trailing spaces not allowed.  no-trailing-spaces
                                    info  - Need to disable some ESLint rules? Learn more here: https://nextjs.org/docs/basic-features/eslint#disabling-rules
2024-02-22T01:07:56.626Z [ERROR]: !!! Build failed
2024-02-22T01:07:56.626Z [INFO]: Please read more about Amplify Hosting's support for SSR frameworks to find if your build failure is related to an unsupported feature: https://docs.aws.amazon.com/amplify/latest/userguide/amplify-ssr-framework-support.html. You may also find this troubleshooting guide useful: https://docs.aws.amazon.com/amplify/latest/userguide/troubleshooting-ssr-deployment.html
2024-02-22T01:07:56.627Z [ERROR]: !!! Error: Command failed with exit code 1
2024-02-22T01:07:56.627Z [INFO]: # Starting environment caching...
2024-02-22T01:07:56.627Z [INFO]: # Environment caching completed
Terminating logging...
 
```

**Comment by emilyemilyemilyemilyemilyemily** [comment]
This is an automated comment I'm testing something. Logs:
```
 
```

**Comment by emilyemilyemilyemilyemilyemily** [comment]
This is an automated comment I'm testing something. Logs:
```
2024-02-22T02:55:55.426Z [INFO]: # Cloning repository: git@github.com:clearviction-devs/clearviction-wa.git
2024-02-22T02:55:57.713Z [INFO]: 
2024-02-22T02:55:57.714Z [INFO]: Cloning into 'clearviction-wa'...
2024-02-22T02:55:58.030Z [INFO]: From https://github.com/clearviction-devs/clearviction-wa
                                 * [new ref]         refs/pull/185/head -> pr-185
                                 Switched to branch 'pr-185'
2024-02-22T02:55:58.143Z [INFO]: Successfully cleaned up Git credentials
2024-02-22T02:55:58.143Z [INFO]: # Checking for Git submodules at: /codebuild/output/src672636271/src/clearviction-wa/.gitmodules
2024-02-22T02:55:58.150Z [INFO]: # Retrieving environment cache...
2024-02-22T02:55:58.195Z [WARNING]: ! Unable to write cache: {"code":"ERR_BAD_REQUEST","message":"Request failed with status code 404"})}
2024-02-22T02:55:58.195Z [INFO]: ---- Setting Up SSM Secrets ----
2024-02-22T02:55:58.195Z [INFO]: SSM params {"Path":"/amplify/dqqxalj4ndxdp/prbif/","WithDecryption":true}
2024-02-22T02:55:58.239Z [WARNING]: !Failed to set up process.env.secrets
2024-02-22T02:56:01.454Z [INFO]: # Node version 18 is available for installation
2024-02-22T02:56:01.571Z [INFO]: # Installing Node version 18
2024-02-22T02:57:01.147Z [INFO]: # Now using Node version 18
2024-02-22T02:57:01.390Z [INFO]: # Retrieving cache...
2024-02-22T02:57:01.422Z [INFO]: # Retrieved cache
2024-02-22T02:57:04.929Z [INFO]: ## Starting Backend Build
                                 ## Checking for associated backend environment...
                                 ## No backend environment association found, continuing...
                                 ## Completed Backend Build
2024-02-22T02:57:04.939Z [INFO]: {"backendDuration": 0}
                                 ## Starting Frontend Build
                                 # Starting phase: preBuild
                                 # Executing command: npm ci
2024-02-22T02:57:05.796Z [WARNING]: npm
2024-02-22T02:57:05.796Z [WARNING]: WARN EBADENGINE Unsupported engine {
                                    npm WARN EBADENGINE   package: '@sanity/webhook@4.0.1',
                                    npm WARN EBADENGINE   required: { node: '>=20.0.0' },
                                    npm WARN
2024-02-22T02:57:05.796Z [WARNING]: EBADENGINE   current: { node: 'v18.19.1', npm: '10.2.4' }
                                    npm WARN EBADENGINE }
2024-02-22T02:57:38.850Z [INFO]: > clearviction-website@2.1.0 prepare
                                 > husky install
2024-02-22T02:57:38.896Z [WARNING]: install command is deprecated
2024-02-22T02:57:38.919Z [INFO]: added 937 packages, and audited 938 packages in 34s
2024-02-22T02:57:38.919Z [INFO]: 173 packages are looking for funding
                                 run `npm fund` for details
2024-02-22T02:57:38.921Z [INFO]: found 0 vulnerabilities
2024-02-22T02:57:38.951Z [INFO]: # Completed phase: preBuild
                                 # Starting phase: build
                                 # Executing command: npm run build
2024-02-22T02:57:39.217Z [INFO]: > clearviction-website@2.1.0 build
                                 > export build=true && next build
2024-02-22T02:57:39.750Z [INFO]: ⚠ No build cache found. Please configure build caching for faster rebuilds. Read more: https://nextjs.org/docs/messages/no-cache
2024-02-22T02:57:39.762Z [INFO]: Attention: Next.js now collects completely anonymous telemetry regarding usage.
2024-02-22T02:57:39.763Z [INFO]: This information is used to shape Next.js' roadmap and prioritize features.
                                 You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
                                 https://nextjs.org/telemetry
2024-02-22T02:57:39.841Z [INFO]: Linting and checking validity of types ...
2024-02-22T02:57:49.936Z [INFO]: Creating an optimized production build ...
2024-02-22T02:58:02.367Z [INFO]: ✓ Compiled successfully
2024-02-22T02:58:02.375Z [INFO]: Collecting page data ...
2024-02-22T02:58:04.782Z [INFO]: Generating static pages (0/65) ...
2024-02-22T02:58:07.096Z [INFO]: Generating static pages (16/65)
2024-02-22T02:58:09.317Z [INFO]: Generating static pages (32/65)
2024-02-22T02:58:10.725Z [INFO]: Generating static pages (48/65)
2024-02-22T02:58:13.029Z [INFO]: ✓ Generating static pages (65/65)
2024-02-22T02:58:27.340Z [INFO]: Finalizing page optimization ...
2024-02-22T02:58:27.345Z [INFO]: Collecting build traces ...
2024-02-22T02:58:30.239Z [INFO]: 
2024-02-22T02:58:30.243Z [INFO]: Route (pages)                                        Size     First Load JS
                                 ┌ ○ /                                                9.67 kB         183 kB
                                 ├   /_app                                            0 B             143 kB
                                 ├ ○ /404                                             182 B           143 kB
                                 ├ ○ /about                                           3.08 kB         161 kB
                                 ├ λ /api/revalidate                                  0 B             143 kB
                                 ├ ● /calculator/[slug] (13849 ms)                    186 kB          366 kB
                                 ├   ├ /calculator/m-offense-dv-1-cont (632 ms)
                                 ├   ├ /calculator/m-circ-main-4-ineligible (541 ms)
                                 ├   ├ /calculator/m-offense-dv-2-cont (430 ms)
                                 ├   ├ /calculator/head-mis-3-cont (329 ms)
                                 ├   ├ /calculator/m-offense-pro-5-ineligible
                                 ├   ├ /calculator/m-offense-pro-6-eligible
                                 ├   ├ /calculator/m-terms-main-3-ineligible
                                 ├   └ [+49 more paths]
                                 ├ ○ /contact                                         4.22 kB         163 kB
                                 ├ ○ /donate                                          4.39 kB         162 kB
                                 ├ ○ /get-involved (861 ms)                           25.5 kB         204 kB
                                 ├   └ css/4df78f2cd73d6b26.css                       1.13 kB
                                 ├ ○ /get-started                                     5.35 kB         167 kB
                                 └ ○ /why-vacate                                      6.29 kB         164 kB
                                 + First Load JS shared by all                        143 kB
                                 ├ chunks/framework-467b11a89995b152.js             45.4 kB
                                 ├ chunks/main-75bdb96a41ba80f4.js                  33.1 kB
                                 ├ chunks/pages/_app-9946092127ad02a6.js            62.2 kB
                                 ├ chunks/webpack-bda1b5b123d36788.js               1.92 kB
                                 └ css/c6b4809f3cddaab8.css                         831 B
2024-02-22T02:58:30.244Z [INFO]: λ  (Server)  server-side renders at runtime (uses getInitialProps or getServerSideProps)
                                 ○  (Static)  automatically rendered as static HTML (uses no initial props)
                                 ●  (SSG)     automatically generated as static HTML + JSON (uses getStaticProps)
2024-02-22T02:58:30.289Z [INFO]: # Completed phase: build
                                 ## Completed Frontend Build
2024-02-22T02:58:30.298Z [INFO]: ## Build completed successfully
2024-02-22T02:58:30.299Z [INFO]: # Starting caching...
2024-02-22T02:58:30.309Z [INFO]: # Creating cache artifact...
2024-02-22T02:58:49.282Z [INFO]: # Created cache artifact
2024-02-22T02:58:49.369Z [INFO]: # Uploading cache artifact...
2024-02-22T02:58:53.841Z [INFO]: # Uploaded cache artifact
2024-02-22T02:58:53.994Z [INFO]: # Caching completed
2024-02-22T02:59:17.260Z [INFO]: # Starting environment caching...
2024-02-22T02:59:17.376Z [INFO]: # Environment caching completed
Terminating logging...
 
```

**Review by emilyemilyemilyemilyemilyemily — APPROVED** [review]

File: components/calculator/misdemeanor/QandAContainer.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import {
   Box, Button, Container, Stack,
 } from '@mui/material';
+// eslint-disable-next-line import/no-unresolved
```
Comment (e-papanicolas) [inline components/calculator/misdemeanor/QandAContainer.tsx:4]: Can you remove this line? It shouldn't be needed with the package being in the package.json file. @shondriane

**Review by e-papanicolas — APPROVED** [review]

**Pull request merged**